diktat-ruleset/src/test/kotlin/org/cqfn/diktat/ruleset/smoke/DiktatSaveSmokeTest.kt

```diff
@@ -77,8 +82,8 @@ class DiktatSaveSmokeTest : DiktatSmokeTestBase() {
                 }
 
                 val saveProcess = processBuilder.start()
-                val saveExitCode = saveProcess.waitFor()
-                softly.assertThat(saveExitCode).describedAs("The exit code of SAVE").isZero
+                val saveProcessResult = saveProcess.waitFor(TEST_TIMEOUT_SECONDS, TimeUnit.SECONDS)
```

Comment:
1. If the external process is hanging, it will remain in the hung state, regardless of the timeout.
2. Since unit tests have just the same time-out, they will be aggressively stopped by JUnit (both versions of `waitFor()` call will get terminated with an `InterruptedException` anyway), and you'll have no `saveProcessResult` anyway. So my opinion here is the time-out just puzzles whomever ir reading the code while adding no value.

Reply:
previous version doesn't work. so I can make waitFor as infinitive to satisfy your request about puzzles